packages/engine/src/avatar/functions/spawnAvatarReceptor.ts

```diff
@@ -121,7 +121,7 @@ export const spawnAvatarReceptor = (spawnAction: typeof WorldNetworkAction.spawn
     createAvatarCollider(entity)
   }
 
-  addComponent(entity, ShadowComponent, { receive: true, cast: true })
+  addComponent(entity, ShadowComponent, { receive: true, cast: true, castDropShadow: true })
```

Comment:
instead of a new property, lets just use `cast` and use drop shadows only when 3d shadows are turned off

Reply:
This works, I just recall @speigg mentioning it would be ideal to be able to use drop shadows together with real shadows. Unless I recall incorrectly of course.